File: kafka-event-emitter/src/main/java/tech/gklijs/emitter/configuration/AxonConfig.java
package tech.gklijs.emitter.configuration;

import com.thoughtworks.xstream.XStream;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class AxonConfig {

    @Bean
    public XStream xStream() {
        XStream xStream = new XStream();

        xStream.allowTypesByWildcard(new String[]{
                "java.util.**",
                "tech.gklijs.api.**"
        });
        return xStream;
    }
}

File: projector/src/main/kotlin/tech/gklijs/projector/AxonConfig.kt
package tech.gklijs.projector

import com.thoughtworks.xstream.XStream
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration


@Configuration
class AxonConfig {

    @Bean
    fun xStream(): XStream? {
        val xStream = XStream()
        xStream.allowTypesByWildcard(
            arrayOf(
                "java.util.**",
                "tech.gklijs.api.**"
            )
        )
        return xStream
    }
}